init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/service/ServiceInvoker.java

```diff
@@ -39,6 +41,7 @@ protected ServiceInvoker(WebClient webClient) {
                     .body(Mono.just(payload), payload.getClass())
                     .retrieve()
                     .bodyToMono(responseExpected)
+                    .retryWhen(Retry.backoff(9, Duration.ofSeconds(5)).maxBackoff(Duration.ofSeconds(30)))
```

Comment:
Consider filtering by server error, not sure, but maybe other errors like 4xx might also get included.

    .retryWhen(Retry.backoff(9, Duration.ofSeconds(5)).maxBackoff(Duration.ofSeconds(30)))
            .filter(this::is5xxServerError));

//...

private boolean is5xxServerError(Throwable throwable) {
    return throwable instanceof WebClientResponseException &&
            ((WebClientResponseException) throwable).getStatusCode().is5xxServerError();
}